### ISSUE TITLE
[SUGGESTION] Add a timeout for WPS connection try

### ISSUE THREAD

**Issue by CoolDude-amber** [drygdryg/OneShot#88]
Sometimes, the script just gets stuck at some step while trying a PIN and I have to abort it myself. I'd recommend adding a 15-20 second timeout WHILE testing the PIN (not between PIN and PIN) 

If that doesn't make sense, ask me for anything

**Comment by CoolDude-amber** [comment]
Note: it might get stuck for hours if I don't check on it

**Comment by october262** [comment]
do you get M1 - M2 - and then warning received timeout occurred 
or something similar to that effect when trying pins ??

**Comment by CoolDude-amber** [comment]
No, it has no warning, let's say I bruteforce for 30 minutes. It says Trying PIN "XXXXXXXX" and it just gets stuck. If I abort the operation and start from that exact same PIN, everything works fine till it suddenly stops again at a different one.
